Firebase/android/app/src/main/java/com/novoda/bonfire/chat/view/MessageView.java

```diff
@@ -28,6 +28,7 @@
 
     public MessageView(Context context, AttributeSet attrs) {
         super(context, attrs);
+        setOrientation(VERTICAL);
```

Comment:
Since you are forcing the orientation, it might be a good idea to use the `super.setOrientation()` instead, override the `setOrientation()` method and only print a log or throw an exception if someone tries to mess with it